CRM/Core/Lock.php

```diff
@@ -145,7 +145,7 @@ public function __destruct() {
    */
   public function acquire($timeout = NULL) {
     if (!$this->_hasLock) {
-      if (self::$jobLog && CRM_Core_DAO::singleValueQuery("SELECT IS_USED_LOCK( '" . self::$jobLog . "')")) {
+      if (self::$jobLog && CRM_Core_DAO::singleValueQuery("SELECT IS_USED_LOCK( '" . $this->_id . "')")) {
```

Comment:
@eileenmcnaughton @totten I suspect this is the line of code that's not quite right... it might be more obvious to you what's wrong though!